connector/src/main/java/org/geysermc/connector/command/defaults/DumpCommand.java

```diff
@@ -48,6 +48,7 @@
     private final GeyserConnector connector;
     private static final ObjectMapper MAPPER = new ObjectMapper();
     private static final String DUMP_URL = "https://dump.geysermc.org/";
+    public static boolean logsDump = false;
```

Comment:
Since this value depends on the instance of DumpCommand, it shouldn't be static (I would also make it private)

I would just add a createLogs boolean to DumpInfo() and LogsInfo()